### ISSUE TITLE
Build instructions?

### ISSUE THREAD

**Issue by mdeguzis** [chaosforgeorg/fpcvalkyrie#2]
How do you compile this? Sorry for being a bit naive, since I have not built pascal projects before. I saw scripts/lua_make.lua, but lua makefile.lua only detects Linux as the OS then exits. If this should be a separate issue, that is fine, but I know this is a very early dump of the code. Maybe the assumption is, like doomrl, the instructions will be posted after the Kickstarter project concludes.

I'm sorry to be "that guy," but someone was bound to ask at some point.

**Comment by epyon** [comment]
FreePascal doesn't have the concept of building libraries, so you basically compile it alongside DoomRL. Will update after KS!

**Comment by subtract1** [comment]
I can give you a quick starter geared for 'I've never dealt with Pascal before'.  It's not a guide but I have built it on Windows (and failed to build on Linux but that looks like a compiler bug).

* Download this and DoomRL.  Put them side by side.  Avoid spaces in directory names.
* Download Lazarus with fpc bundled in (lazarus-1.6.2-fpc-3.0.0-win64 for most people)
* Install Lazarus.
* Open the DoomRL project (doomrl/src/doomrl.lpi)
* Push the 'build' button
* Cross your fingers

**Comment by mdeguzis** [comment]
@subtract1 , thank you, i'll check out the IDE. Is there a CLI-equivalent?

>Cross your fingers

I like the way you think. Anyway, my end goal is to host packages for SteamOS/Debian/Ubuntu when time allows (private repos, not upstream-sanctioned).

**Comment by subtract1** [comment]
@mdeguzis 

Yes you can use fpc directly.  And yes it is in most distros but I was NOT able to compile it myself--got an internal error.  Internal errors in fpc as I understand them are compiler bugs; can't do much about that.

**Comment by epyon** [comment]
Closing as this is very not up to date compared to both the master branch readme, as well as development readme!
